Vienna/Sources/Shared/ImageAndTextCell.m

```diff
@@ -325,32 +325,26 @@ -(void)selectWithFrame:(NSRect)aRect inView:(NSView *)controlView editor:(NSText
 	[super selectWithFrame:aRect inView:controlView editor:textObj delegate:anObject start:selStart length:selLength];
 }
 
--(NSArray*)accessibilityAttributeNames
-{
-    static NSArray * attributes = nil;
-    if (!attributes)
-    {
-        NSSet * set = [NSSet setWithArray:[super accessibilityAttributeNames]];
-        attributes = [set setByAddingObject:NSAccessibilityDescriptionAttribute].allObjects;
+- (NSString *)accessibilityLabel {
+    NSMutableArray *bits = [NSMutableArray arrayWithCapacity:3];
```

Comment:
is `bits` the best name to use here? How about something like `accessibilityLabelData` or similar?

Reply:
I didn’t change this part of the code. 😅

Reply:
heh OK